agent/deployment_stages/common.py

```diff
@@ -119,7 +119,7 @@ def find_healthchecks(check_type, archive_dir, appspec, logger):
 
 def wrap_script_command(script, platform):
     if platform == 'windows':
-        return 'powershell.exe -NonInteractive -NoProfile -ExecutionPolicy RemoteSigned -Command {0}'.format(script)
+        return 'powershell.exe -NonInteractive -NoProfile -ExecutionPolicy RemoteSigned -Command "{0}"'.format(script)
```

Comment:
This may be a case of bad naming in the `wrap_script_command` function. The value is it passed is the absolute path of the script.